### PR TITLE
Create karatsuba.py

### PR DESCRIPTION
**Pull request by rishabh0098**
Added karatsuba algorithm for multiplication of two numbers

### CODE REVIEW

File: maths/karatsuba.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+def karatsuba(a, b):
+    if len(str(a)) == 1 or len(str(b)) == 1:
+        return (a * b)
+    else:
+        m1 = max(len(str(a)), len(str(b)))
+        m2 = m1 // 2
+
+        a1 = a // 10**(m2)
+        a2 = a % 10**(m2)
```
Comment (cclauss) [inline maths/karatsuba.py]: __a1, a2 = divmod(a, 10**m2)__

File: maths/karatsuba.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        b1 = b // 10**(m2)
+        b2 = b % 10**(m2)
+
```
Comment (cclauss) [inline maths/karatsuba.py]: __b1, b2 = divmod(b, 10**m2)__

File: maths/karatsuba.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
+""" Multiply two numbers using Karatsuba algorithm """
+
+def karatsuba(a, b):
```
Comment (cclauss) [inline maths/karatsuba.py]: """
>>> karatsuba(15463, 23489) == 15463 * 23489
True
"""

Comment (rishabh0098) [reply maths/karatsuba.py]: done :+1:

File: maths/karatsuba.py (hunk 2 above)
Comment (rishabh0098) [reply maths/karatsuba.py]: done :+1:

File: maths/karatsuba.py [hunk 4]
```diff
@@ -1,16 +1,19 @@
 """ Multiply two numbers using Karatsuba algorithm """
 
 def karatsuba(a, b):
+    """
+    >>> karatsuba(15463, 23489) == 15463 * 23489
+    True
+    >>> karatsuba(3, 9) == 3 * 9
+    """
```
Comment (cclauss) [inline maths/karatsuba.py]: True

**Comment by cclauss** [comment]
https://travis-ci.org/TheAlgorithms/Python/builds/595051157#L502

**Pull request merged**